core/examples/abuse/Main.hs

```diff
@@ -1,167 +0,0 @@
-{-# LANGUAGE BangPatterns       #-}
```

Comment:
Why is this file deleted?

Reply:
`abuse` used `Mockable` & `Production`, and in trying to remove `Mockable` from it, I realized `abuse` was very bitrotten and contained imports of modules deleted from `cardano-sl` months ago. It seemed to me that the cost of updating the code would be significant, and that deleting it might be the best call. @avieth agreed with this, and I believe he was the last one to touch `abuse`.

I put a comment in the [git commit](https://github.com/input-output-hk/cardano-sl/pull/3285/commits/9007b703e09a3f8b1076f9b232b29418bdfd4973) but should've also written an explanatory comment on the PR.